assets/js/blocks/cart-checkout/checkout/utils.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Get the number of packages in a shippingRates array.
+ *
+ * @param {Array} shippingRates Shipping rates and packages array.
+ */
+export const getShippingRatesPackageCount = ( shippingRates ) => {
+	return Object.keys( shippingRates ).length;
```

Comment:
I just merged it, but shouldn't this be:
```suggestion
	return shippingRates.length;
```
`shippingRates` is an array, not an object.

I will create a follow-up PR.

Reply:
I tested and `Object.keys()` works for arrays too. In any case, I removed it in #2153.